kv/label.go

```diff
@@ -311,7 +311,8 @@ func (s *Service) PutLabel(ctx context.Context, l *influxdb.Label) error {
 	})
 }
 
-func (s *Service) createUserResourceMappingForOrg(ctx context.Context, tx Tx, orgID influxdb.ID, resID influxdb.ID, resType influxdb.ResourceType) error {
+// CreateUserResourceMappingForOrg can be made private once URMs are removed from the Label Service
+func (s *Service) CreateUserResourceMappingForOrg(ctx context.Context, tx Tx, orgID influxdb.ID, resID influxdb.ID, resType influxdb.ResourceType) error {
```

Comment:
```suggestion
func (s *Service) CreateUserResourceMappingForOrg(ctx context.Context, orgID influxdb.ID, resID influxdb.ID, resType influxdb.ResourceType) error {
	return s.kv.Update(ctx, func(tx Tx) error {
		return s.createUserResourceMappingForOrg(ctx, tx, orgID, resID, resType)
	}
}

func (s *Service) createUserResourceMappingForOrg(ctx context.Context, tx Tx, orgID influxdb.ID, resID influxdb.ID, resType influxdb.ResourceType) error {
```

This was you can use the lower-case create in the kv package. But externally you can consume the exported version which implicitly creates an update transaction.